samples/typescript/units/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "units-sample",
+  "version": "1.0.0",
+  "private": true,
+  "license": "Apache-2.0",
+  "scripts": {
+    "tsc": "tsc",
+    "start": "yarn tsc && node build/quickstart.js"
+  },
+  "dependencies": {
+    "@cognite/auth-wrapper": "^1",
+    "@cognite/sdk": "^7",
+    "@cognite/units": "^1.0.2"
```

Comment:
maybe it a good idea to change to "@cognite/units": "^1"

Reply:
I guess this came in from semantic version format.